chef/data_bags/crowbar/bc-template-glance.schema

```diff
@@ -76,6 +76,7 @@
             "rbd": {
               "type": "map", "required": true, "mapping": {
                 "store_ceph_conf": { "type": "str", "required": true },
+                "store_admin_keyring": { "type": "str", "required": true },
```

Comment:
Ok., I guess I can live with that. But making this a required attribute creates the wrong impression that ""store_admin_keyring" must always be present and the admin keyring  is always required even when an external ceph cluster is used.